npc/re/mobs/dungeons/rockridge.txt

```diff
@@ -0,0 +1,23 @@
+//===== rAthena Script =======================================
+//= Rock Ridge Monsters Spawn Script
+//===== Additional Comments: =================================
+//= 1.0 First version. [Capuche]
+//============================================================
+
+// Rock Ridge Mine
+rockmi1,0,0	monster	Thief Bug	1051,3
+rockmi1,0,0	monster	Gaster	3740,18
+rockmi1,0,0	monster	Elite Buffalo Bandit	3747,3
+rockmi1,0,0	monster	Elite Buffalo Bandit	3748,8
+rockmi1,0,0	monster	Elite Buffalo Bandit	3749,6
+rockmi1,0,0	monster	Spider Chariot	3741,1
```

Comment:
This its a boss monter, so:
-rockmi1,0,0	monster	Spider Chariot	3741,1
+rockmi1,0,0,0,0	boss_monster	Spider Chariot	3741,1,7200000,600000,1

Font: https://www.divine-pride.net/database/monster/3741